HoneyBadgers.Logic/User.cs

```diff
@@ -13,5 +13,7 @@ public class User
         public string LastName { get; set; }
         public string Email { get; set; }
         public List<Movie> Movies { get; set; }
```

Comment:
myślała że będziemy bazować na tej liście Movies, ale jeśli tego nie używasz to usuń